zio-http/src/test/scala/zhttp/http/HttpSpec.scala

```diff
@@ -136,11 +136,47 @@ object HttpSpec extends DefaultRunnableSpec with HExitAssertion {
             val actual = (a ++ b).execute(2)
             assert(actual)(isSuccess(equalTo("B")))
           } +
+          test("should resolve third") {
```

Comment:
- Move tests for `++` into the `combine` suite.
- For testing the `++` use `succeed` and `empty` to build Https, instead of `collect`